src/web/services/feedback-questions.service.ts

```diff
@@ -271,20 +270,33 @@ export class FeedbackQuestionsService {
           showGiverNameTo: [FeedbackVisibilityType.INSTRUCTORS],
           showRecipientNameTo: [FeedbackVisibilityType.INSTRUCTORS, FeedbackVisibilityType.RECIPIENT],
         };
+
       case FeedbackQuestionType.NUMSCALE:
+
         return {
           questionBrief: '',
           questionDescription: '',
 
           questionType: FeedbackQuestionType.NUMSCALE,
-          questionDetails: {
-            minScale: 1,
-            maxScale: 5,
-            step: 1,
-            questionType: FeedbackQuestionType.NUMSCALE,
-            questionText: '',
-          } as FeedbackNumericalScaleQuestionDetails,
+          questionDetails: DEFAULT_NUMSCALE_QUESTION_DETAILS(),
+          giverType: FeedbackParticipantType.STUDENTS,
+          recipientType: FeedbackParticipantType.OWN_TEAM_MEMBERS,
+
+          numberOfEntitiesToGiveFeedbackToSetting: NumberOfEntitiesToGiveFeedbackToSetting.UNLIMITED,
+
+          showResponsesTo: [FeedbackVisibilityType.INSTRUCTORS, FeedbackVisibilityType.RECIPIENT],
+          showGiverNameTo: [FeedbackVisibilityType.INSTRUCTORS],
+          showRecipientNameTo: [FeedbackVisibilityType.INSTRUCTORS, FeedbackVisibilityType.RECIPIENT],
+        };
+
+      case FeedbackQuestionType.MCQ:
+
+        return {
+          questionBrief: '',
+          questionDescription: '',
 
+          questionType: FeedbackQuestionType.MCQ,
+          questionDetails: DEFAULT_MCQ_QUESTION_DETAILS(),
```

Comment:
You mean to change it here?

Reply:
Exactly, that way you (and anyone reading the code) know exactly the value being set.

Reply:
Right, I was just skeptical because I thought Default structure had to be used to initialize question details and maintain consistency across the questions